extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -179,3 +208,69 @@ function setup_tracks_callback( \SimplePie_Item $episode ) {
 
 	return $track;
 }
+
+/**
+ * Build an array with CSS classes and inline styles defining the colors
+ * which will be applied to player either in the editor-canvas
+ * as well as in the front-end side.
+ *
+ * @param  array $attributes block attributes.
+ * @return array Colors CSS classes and inline styles.
+ */
+function get_colors( $attributes ) {
+	$colors = array(
+		'css_classes'      => array(),
+		'css_text_classes' => array(),
+		'style'            => '',
+		'class_name'       => '',
+		'text_class_name'  => '',
+	);
+
+	// Text color.
+	$has_named_text_color  = isset( $attributes['textColor'] );
+	$has_custom_text_color = isset( $attributes['customTextColor'] );
+
+	// If has text color.
+	if ( $has_custom_text_color || $has_named_text_color ) {
+		// Add has-text-color class.
+		$colors['css_classes'][]      = 'has-text-color';
+		$colors['css_text_classes'][] = 'has-text-color';
```

Comment:
Trying to understand why both of these are necessary. I think `$colors['css_classes'][]      = 'has-text-color';` maybe doesn't need to be there?

Reply:
Very valid comment. We need to know the CSS classes will be applied only for text. It will be used for the podcast title, applied in the `<ol><li>` elements.
For this reason, is why added this extra index to the `$colorsa array.